docs/docs/cmd/spo/navigation/navigation-node-list.md

```diff
@@ -39,7 +39,13 @@ m365 spo navigation node list --webUrl https://contoso.sharepoint.com/sites/team
     ```json
     [
       {
+        "AudienceIds": null,
```

Comment:
Could we add a value for this property as well so it's clear what people can expect.